src/api/publish.js

```diff
@@ -17,6 +22,10 @@ import make from './make';
  * @property {string} [tag=packageJSON.version] The string to tag this release with
  * @property {Array<string>} [publishTargets=[github]] The publish targets
  * @property {MakeOptions} [makeOptions] Options object to passed through to make()
+ * @property {string} [outDir=`${dir}/out`] The path to the directory containing generated distributables
+ * @property {boolean} [dryRun=false] Whether or not to generate dry run meta data and not actually publish
+ * @property {boolean} [dryRunResume=false] Whether or not to attempt to resume a previously saved dryRun and publish
+ * @property {Object} [makeResults=null] Provide results from make so that the publish step doesn't run make itself
```

Comment:
Seems like this Object should be explained a bit more.

Reply:
Linked it to the `MakeResult` declaration